sidebars.js

```diff
@@ -248,7 +248,11 @@ const sidebars = {
             type: "doc",
             id: "concepts/pdp/overview",
           },
-          items: ["concepts/pdp/configuration", "concepts/pdp/factdb"],
+          items: [
+            "concepts/pdp/configuration",
+            "concepts/pdp/factdb",
```

Comment:
I think we need to delete this docs

Reply:
Opened a ticket for it
https://linear.app/permit/issue/PER-12068/remove-docs-for-factdb